README-Chinese.md

```diff
@@ -0,0 +1,115 @@
+<h1 align="center">Auto PY to EXE</h1>
+<p align="center">一个.py到.exe的转换器，使用简单的图形界面和Python中的<a href="https://www.pyinstaller.org/">PyInstaller</a></p>
+
+<p align="center">
+    <img src="https://nitratine.net/posts/auto-py-to-exe/feature.png" alt="Empty interface">
+</p>
+
+<p align="center">
+    <a href="https://pypi.org/project/auto-py-to-exe/"><img src="https://img.shields.io/pypi/v/auto-py-to-exe.svg" alt="PyPI Version"></a>
+    <a href="https://pypi.org/project/auto-py-to-exe/"><img src="https://img.shields.io/pypi/pyversions/auto-py-to-exe.svg" alt="PyPI Supported Versions"></a>
+    <a href="https://pypi.org/project/auto-py-to-exe/"><img src="https://img.shields.io/pypi/l/auto-py-to-exe.svg" alt="License"></a>
+    <a href="https://pepy.tech/project/auto-py-to-exe"><img src="https://static.pepy.tech/badge/auto-py-to-exe/month" alt="Downloads Per Month"></a>
+    <a href="https://pyinstaller.readthedocs.io/en/stable/requirements.html"><img src="https://img.shields.io/badge/platform-windows%20%7C%20linux%20%7C%20macos-lightgrey" alt="Supported Platforms"></a>
+</p>
+
+### To read README in English, press [here](https://github.com/jiangzhe11/auto-py-to-exe/blob/master/README.md)
```

Comment:
I think this will work better

```suggestion
### To read README in English, press [here](./README.md)
```

This means all forks will be linked to their own repo also (if that is desired).